### PR TITLE
add torch_blade::FakeQuant

### LINKED ISSUE #568: TensorFlow/PyTorch fake_quant custom ops and their corresponding implementation in TF/PyTorch Dialect.
Subtask of #567

### PR DESCRIPTION
**Pull request by qiuxiafei**

### CODE REVIEW

File: pytorch_blade/bazel_build.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def __init__(self, *args, **kwargs):
         # note: If we use backend (such as trt) to do calibration and get the
         # quantization engine, this limitation should not be set.
         is_enable_quantization = \
-            self.torch_major_version == 1 and 8 <= self.torch_minor_version < 12
+            self.torch_major_version == 1 and self.torch_minor_version >= 8
```
Comment (wyzero) [inline pytorch_blade/bazel_build.py:130]: What if quantizedInput > 1

File: pytorch_blade/src/quantization/fake_quant.cpp [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+at::Tensor FakeQuant::forward(
+    const at::Tensor& input,
+    const at::Tensor& scale,
+    const at::Tensor& zero_point) {
+  if (per_channel_) {
+    // torch version 1.8.x and 1.9.x require Long type for zero_point, while
+    // higher versions require Int/Float/Double typle.
+#if PYTORCH_MAJOR_VERSION == 1 && PYTORCH_MINOR_VERSION < 10
+    at::Tensor zero_point_new = zero_point.to(at::ScalarType::Long);
+#else
```
Comment (wyzero) [inline pytorch_blade/src/quantization/fake_quant.cpp:63]: What if PYTORCH_MAJOR_VERSION < 1

**Review by wyzero — COMMENTED** [review]
LGTM

File: pytorch_blade/src/quantization/fake_quant.cpp (hunk 2 above)
Comment (qiuxiafei) [reply pytorch_blade/src/quantization/fake_quant.cpp:64]: fixed

**Review by wyzero — APPROVED** [review]
LGTM

**Pull request merged**